Makefile

```diff
@@ -2,10 +2,12 @@
 # Override in order to customize
 IMAGE ?= kabanero-operator:latest
 REGISTRY_IMAGE ?= kabanero-operator-registry:latest
+COLLECTION_CTRLR_IMAGE ?= kabanero-collection-operator:latest
```

Comment:
@dacleyra Please have a look at this makefile and let us know what additional bits you need for your integration tests, and also when using the internal registry.